sql-plugin/src/main/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -724,16 +716,16 @@ trait GpuDivModLike extends CudfBinaryArithmetic {
     if (failOnError) {
       withResource(makeZeroScalar(rhs.getBase.getType)) { zeroScalar =>
         if (rhs.getBase.contains(zeroScalar)) {
-          divByZeroError()
+          throw RapidsErrorUtils.divByZeroError(origin)
         }
         if (checkDivideOverflow && isDivOverflow(lhs, rhs)) {
-          divOverflowError()
+          throw RapidsErrorUtils.divOverflowError(origin)
         }
         super.doColumnar(lhs, rhs)
       }
     } else {
       if (checkDivideOverflow && isDivOverflow(lhs, rhs)) {
-        divOverflowError()
+          throw RapidsErrorUtils.divOverflowError(origin)
```

Comment:
Indentation seems off here
```suggestion
        throw RapidsErrorUtils.divOverflowError(origin)
```